src/pages/ShopDetail/ReviewWritingPage/ReviewWritingPage.scss

```
@@ -0,0 +1,67 @@
+.reviewWritingPage {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  font-size: 16px;
+}
+
+.reviewWritingPageInner {
+  width: 678px;
+  //   border: 1px solid black;
+
+  .reviewWritingPageTitle {
+    margin-bottom: 25px;
+    //   border: 1px solid blue;
+    .restaurantName {
+      color: #ff7100;
+      word-break: break-word;
+      font-size: 40px;
+    }
+    .subMessage {
+      color: rgba(79, 79, 79, 0.9);
+    }
+  }
+  .reviewBox {
+    border: 1px solid rgba(79, 79, 79, 0.5);
+    padding: 20px;
+
+    .reviewGrade {
+      display: flex;
+    }
+    .reviewMessage {
+      overflow-wrap: break-word;
+      height: 150px;
+    }
+    textarea {
+      height: 150px;
+      width: 100%;
+      color: rgba(79, 79, 79, 0.5);
+      margin-top: 20px;
+    }
+    textarea::placeholder {
+      color: rgba(79, 79, 79, 0.4);
+    }
```

tag selector는 다른 요소들에 영향을 받을, 미칠 여지가 많기 때문에 추후 유지보수에 악영향을 미칠 수 있습니다. className을 부여해주셨는데, class selector 사용해주세요. 또한, 다음과 같이 작성할 수 있습니다.

```suggestion
    textarea {
      height: 150px;
      width: 100%;
      color: rgba(79, 79, 79, 0.5);
      margin-top: 20px;
      
      &::placeholder {
            color: rgba(79, 79, 79, 0.4);
      }
    }
```

---

수정했습니다 !